test/e2e/stat.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
Do we want to store this file in git? or it's just a temporary testing script?

Reply:
it will be removed